lib/backend/k8s/custom/globalbgpconfig.go

```diff
@@ -21,23 +21,24 @@ import (
 	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-type GlobalBgpConfigSpec struct {
-	Name  string `json:"name"`
-	Value string `json:"value"`
-}
-
 type GlobalBgpConfig struct {
 	metav1.TypeMeta `json:",inline"`
-	Metadata        metav1.ObjectMeta `json:"metadata"`
+	Metadata        metav1.ObjectMeta   `json:"metadata"`
+	Spec            GlobalBgpConfigSpec `json:"spec"`
+}
 
-	Spec GlobalBgpConfigSpec `json:"spec"`
+type GlobalBgpConfigSpec struct {
+	// The reason we have Name field in Spec is because k8s metadata
+	// name field makes the string lowercase, so Name field in Spec is
```

Comment:
rather than "make the string lowercase", "requires the string to be lowercase" is more accurate :)